### PR TITLE
Tests for create branch

### PR DESCRIPTION
**Pull request by SilkovAlexander**

### CODE REVIEW

File: v2_x/git-remote-gosh/src/git_helper/push/push_diff.rs [hunk 1]
```diff
@@ -291,10 +290,8 @@ pub async fn push_new_branch_snapshot(
     original_content: &[u8],
 ) -> anyhow::Result<()> {
     tracing::trace!("push_new_branch_snapshot: remote_network={remote_network}, dao_addr={dao_addr}, repo_addr={repo_addr}, commit_id={commit_id}, branch_name={branch_name}, file_path={file_path}");
-    let content: Vec<u8> = ton_client::utils::compress_zstd(original_content, None)?;
-    tracing::trace!("compressed to {} size", content.len());
 
-    let (content, ipfs) = if is_going_to_ipfs(&content) {
+    let (content, ipfs) = if is_going_to_ipfs(original_content) {
```
Comment (SilkovAlexander) [inline v2_x/git-remote-gosh/src/git_helper/push/push_diff.rs:305]: Do we store uncompressed data in ipfs?

File: v2_x/git-remote-gosh/src/git_helper/push/mod.rs [hunk 2]
```diff
@@ -279,6 +282,69 @@ where
...
+        let walk = from
+            .ancestors()
+            .all()?
+            .map(|e| e.expect("all entities should be present"))
+            .into_iter();
+
+        let accounts: Vec<_> = walk.map(|a| a.to_string()).collect();
+
+        let query = r#"query($accounts: [String]!) {
+            accounts(filter: {
```
Comment (SilkovAlexander) [inline v2_x/git-remote-gosh/src/git_helper/push/mod.rs:283]: Accounts can be not sorted here

File: v2_x/git-remote-gosh/src/git_helper/push/push_diff.rs (hunk 1 above)
Comment (sauin) [reply v2_x/git-remote-gosh/src/git_helper/push/push_diff.rs:305]: Yea

File: v2_x/git-remote-gosh/src/git_helper/push/mod.rs (hunk 2 above)
Comment (sauin) [reply v2_x/git-remote-gosh/src/git_helper/push/mod.rs:283]: It definitely won't be done, but the subsequent search is performed over a sorted list of commits

**Comment by SilkovAlexander** [comment]
Please add test with snapshot deletion

**Review by andrew-kurochkin — APPROVED** [review]

**Pull request merged**